.grit/patterns/mui4_to_mui5.md

```diff
@@ -0,0 +1,71 @@
+## title: Convert Rename theme.palette.type In MUI v4 To MUI v5
+
+This pattern Rename theme.palette.type in MUI v4 to MUI v5
+
+tags: #react, #migration, #complex
+
+```grit
+engine marzano(0.1)
+language js
+
+`createTheme($theme)` where {
+  $theme <: contains `palette: { $palette }`,
+  $palette <: contains `type: $arg` => `mode: $arg`
+}
+```
+
+## Rename type to mode for palette > MUI v4 to MUI v5
```

Comment:
You don't need to include the context on every test name. The document structure already captures that.

```suggestion
## Rename type to mode for palette
```